buildSrc/src/main/kotlin/org/gradle/plugins/compile/AvailableJavaInstallations.kt

```diff
@@ -0,0 +1,62 @@
+package org.gradle.plugins.compile
+
+import org.gradle.api.JavaVersion
+import org.gradle.internal.jvm.Jvm
+import org.gradle.jvm.toolchain.internal.JavaInstallationProbe
+import org.gradle.jvm.toolchain.internal.LocalJavaInstallation
+import java.io.File
+
+class DefaultJavaInstallation(val current: Boolean) : LocalJavaInstallation {
+    private lateinit var name: String
```

Comment:
:o: Let's put the `private` modifier on its own line, like we do elsewhere:
```kotlin
private
lateinit var name: String
```

Reply:
Done. Do we have something like checkstyle for Kotlin, so we would get automatic checks for these kinds of problems?

Reply:
Not yet, it's on the radar though: https://github.com/gradle/kotlin-dsl/issues/677